docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: "3.3"
+
+networks:
+  default:
+
+services:
+  backend:
+    # build: 
+    #   context: backend
+    #   dockerfile: Dockerfile
+    image: 068992353948.dkr.ecr.us-east-2.amazonaws.com/mips-backend-staging
```

Comment:
Can this also be a var? At least the account ID.

Reply:
Removed